### DISCUSSION TITLE
Unable to Start the Service - 3Node cluster

### DISCUSSION THREAD

**Discussion by rohitpals** [minio/minio#20230] [Q&A]
Hi All,

We have installed MinIO on 3 Nodes with a 3TB Disk on each node.
We are able to open individual Node UI, when we add the 
MINIO_VOLUMES="/data1/minio"

**Also, how 3 cluster nodes will talk to each other, can we see that in UI? Where to configure it?**

**/etch/hosts**
```
10.65.55.61 mum-s3-node-01
10.65.55.62 mum-s3-node-02
10.65.55.63 mum-s3-node-03

```
**/etc/default/minio**

```
MINIO_VOLUMES="http://mum-s3-node-0{1...3}:9000/data{1...3}/minio"
MINIO_OPTS="--console-address :9001"
MINIO_ROOT_USER=minioadmin
```

**Attached disk to each node:**
```
/dev/vdb1       3.0T   22G  3.0T   1% /data1
/dev/vdb1       3.0T   22G  3.0T   1% /data2
/dev/vdb1       3.0T   22G  3.0T   1% /data3

```
**Also getting errors in service status -**

```
root@mum-s3-node-03:/home/ubuntu# service minio status
● minio.service - MinIO
     Loaded: loaded (/lib/systemd/system/minio.service; disabled; vendor preset: enabled)
     Active: active (running) since Thu 2024-08-08 09:23:50 UTC; 1h 47min ago
       Docs: https://min.io/docs/minio/linux/index.html
   Main PID: 1883 (minio)
      Tasks: 22
     Memory: 171.6M
        CPU: 20.933s
     CGroup: /system.slice/minio.service
             └─1883 /usr/local/bin/minio server --console-address :9001 http://mum-s3-node-0{1...3}:9000/data{1...3}/minio

Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-01:9000/data1/minio: Get "http://mu>
Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-02:9000/data1/minio: <nil>, will be retr>
Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-03:9000/data1/minio: <nil>, will be retr>
Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-01:9000/data2/minio: Get "http://mu>
Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Waiting for a minimum of 4 drives to come online (elapsed 1h47m6s)
lines 1-21/21 (END)
```

**Comment by harshavardhana** [comment]
> Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-01:9000/data1/minio: Get "http://mu>
> Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-02:9000/data1/minio: <nil>, will be retr>
> Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-03:9000/data1/minio: <nil>, will be retr>
> Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Unable to connect to http://mum-s3-node-01:9000/data2/minio: Get "http://mu>
> Aug 08 11:11:41 mum-s3-node-03 minio[1883]: INFO: Waiting for a minimum of 4 drives to come online (elapsed 1h47m6s)
> lines 1-21/21 (END)

Paste the whole log, use journalctl -u minio
